src/libraries/System.Runtime.InteropServices.JavaScript/tests/System.Runtime.InteropServices.JavaScript.Legacy.UnitTests/System.Runtime.InteropServices.JavaScript.Legacy.Tests.csproj

```diff
@@ -8,6 +8,7 @@
     <WasmEnableLegacyJsInterop Condition="'$(WasmEnableLegacyJsInterop)' == ''">true</WasmEnableLegacyJsInterop>
     <DefineConstants Condition="'$(WasmEnableLegacyJsInterop)' == 'false'">$(DefineConstants);DISABLE_LEGACY_JS_INTEROP</DefineConstants>
     <Scenario>WasmTestOnBrowser</Scenario>
+    <_XUnitBackgroundExec>false</_XUnitBackgroundExec>
```

Comment:
we should explain somewhere why you would want to set this to false, so someone coming by later understands why it's false for some csprojs and not others